scripts.js

```diff
@@ -1,61 +1,74 @@
 /* eslint-disable no-use-before-define */
```

Comment:
- [ ] Great job so far, kindly don't disable linters, if issues come up let's try to fix them.